oaknational/Oak-Web-Application
# fix: [LESQ-64] change copy on newsletter form

## Description

- Changed the number of users referenced on the newsletter form

## How to test

1. Go to the deployment
2. See wherever the newsletter form is rendered, 100k instead of 80,000

## Checklist
